airbyte-integrations/connectors/source-smartsheets/acceptance-test-config.yml

```diff
@@ -2,9 +2,13 @@ connector_image: airbyte/source-smartsheets:dev
 tests:
   spec:
     - spec_path: "source_smartsheets/spec.json"
+      backward_compatibility_tests_config:
+        disable_for_version: "0.1.14"
```

Comment:
Can you share more about what the user experience will be with this change?  If they will need to re-authenticate, this is a breaking change and should be `v1.0.0`

Reply:
Re-authentication will be needed for all users, as spec file structure will be changed, should I bump `version` to `1.0.0`?

Reply:
If that's the case then yes, this should be a breaking change and need to be communicated to users.

However, in other oauth implementations we have maintained compatibility with older configs by still looking for an access token in the old path so we don't break existing connections. Could we do something similar here?

Reply:
version updated